README.md

```diff
@@ -396,6 +396,259 @@ public CloudTasksClient getService() throws IOException {
 }
 ```
 
+## Retries
+Client Libraries use retries to handle unexpected, transient failures (i.e. Server temporarily unavailable).
+Multiple attempts, hopefully, will result in a successful response from the server.
+
+By default, retries are configured by the Cloud Service. The configured retry parameters are defined per RPC.
+A service *may* choose to only enable retries for a subset of RPCs. It is possible that for a single service, 
+each RPC is configured differently.
+
+### Google Cloud Client Library Retry Concepts
+Enabling retries allow an RPC multiple attempts to try and achieve a successful call. A successful call
+is a response from a server that returns an `OK` Status Code (from gRPC) or a `2xx` Status Code (from HttpJson).
+
+An RPC will be retried when _both_ of the following scenarios occur:
+- Non-successful status code is received by the library and the status code is marked as retryable*
+- An RPC invocation exceeds the individual RPC bounds, but still falls within total RPC bounds**
+
+Note: If only one (or neither) of the scenarios above are true, then the RPC will not be retried.
+i.e. If the total timeout has not been exceeded, but the latest attempt receives a non-retryable status code.
+
+*The client library will mark a status code as retryable internally. It is marked as retryable if the response's
+status code matches with any of an RPC's configure retryable status codes.
+
+**When configuring the RPC bounds, you may configure the bounds for each attempt as well as the
+total RPC's bounds. The retry algorithm will ensure that an individual attempt's bounds falls within
+the total RPC's bounds.
+
+#### Configurable Retry Params
+- Retry Status Code: Set of failure status codes to retry on
+- Retry Time/ Attempt Bounds: Configurable retry settings (via RetrySettings class) to define the retry bounds
+
+#### RetrySettings Configurable Params
+- Max Attempts: The maximum number of attempts to perform. If this value is greater than 0, and the number of attempts reaches this limit, the logic will give up retrying even if the total retry time is still lower than TotalTimeout.
```

Comment:
These info already exist in Javadocs, I don't think we should repeat them here, a link should be good enough IMO.

Reply:
Sounds good. I'll remove.